tasks/main.yml

```diff
@@ -2,23 +2,33 @@
 - name: Include OS-specific variables.
   include_vars: "{{ ansible_os_family }}.yml"
 
-- name: Set the correct timezone.
+- name: Configure timezone (For Ansible 2.2+)
+  timezone:
+    name: "{{ ntp_timezone }}"
+  when: ansible_version.full >= "02.02.00.00"
```

Comment:
Can you just switch to the new method (using `timezone` module) instead of having these two tasks? Just make sure to edit `meta/main.yml` and bump the minimum Ansible version to 2.2.